Climate_police/tests/hw7-mao/pollution_map.py

```diff
@@ -0,0 +1,44 @@
+import plotly.offline as py
+py.init_notebook_mode()
+
+# import pandas as pd
+
+from pre_process import pre_process
+
+def pollution_map(df, source, year):
```

Comment:
Exactly, we can use drop down menus to choose pollutant source, and use sliders to change the year.